### PR TITLE
Added cli option to specify fuel environment

### PR DESCRIPTION
**Pull request by rob-mccann**
I noticed this might be rewritten in the future somehow, but here's what I'll be using in the meantime.

Usage:
oil refine migrate up --env=stage

### CODE REVIEW

File: classes/command.php [hunk 1]
```diff
@@ -23,6 +23,13 @@ class Command
 {
 	public static function init($args)
 	{
+		//set up the environment
+		$env = \Cli::option('env', false);
+		if( $env )
```
Comment (philsturgeon) [inline classes/command.php]: Picky, but this should be `if ($env)`.

Comment (rob-mccann) [inline classes/command.php]: Fixed :)

File: classes/command.php [hunk 2]
```diff
@@ -23,6 +23,13 @@ class Command
 {
 	public static function init($args)
 	{
+		//set up the environment
+		$env = \Cli::option('env', false);
```
Comment (philsturgeon) [inline classes/command.php]: You don't need to specify `false` here as your if statement will work identically with `null` which is the default default.

Just use:

```
if (($env = \Cli::option('env')))
{
     \Fuel::$env = constant('\Fuel::'. strtoupper($env)) ?: \Fuel::DEVELOPMENT;
}
```

**Comment by WanWizard** [comment]
@philsturgeon any more comments?

**Comment by philsturgeon** [comment]
Thanks for the poke @WanWizard.

@unforeseen I don't get an alert when you push, so please just throw a comment in saying "Did it!" and we see that :)

**Comment by rob-mccann** [comment]
Did it :-)

**Pull request merged**